static/src/javascripts/projects/common/modules/commercial/templates/acquisitions-epic-control.js

```diff
@@ -1,8 +1,20 @@
 // @flow
 import { appendToLastElement } from 'lib/array-utils';
 
+const buildFooter = (footer: string[]): string =>
+    `<div class="contributions__epic-footer">
+        ${footer
+            .map(line => {
+                const firstSpaceIndex = line.trim().indexOf(' ');
+                const firstWord = line.substring(0, firstSpaceIndex);
+                const remainder = line.substring(firstSpaceIndex);
+                return `<h2><span class="contributions__epic-footer-blue">${firstWord}</span>${remainder}</h2>`;
```

Comment:
You should probably construct this with `classNameWithModifiers`,  i,e `classNameWithModifiers("contributions__epic-footer", ['blue'])`

Reply:
looks like we don't have a `classNameWithModifiers` here (we've defined our own in support-frontend)

Reply:
oh wow! Well it's a super awesome function that would be a great addition to the codebase, you could add it in! 

Or just do it manually, your call